src/appmixer/mcpservers/upstash_context7_mcp/lib.js

```
@@ -0,0 +1,75 @@
+module.exports = {
+
+    mcpCall: async (context, env, method, args) => {
+
+        const clientName = `mcp-client:${context.flowId}:${context.componentId}:${Math.random().toString(36).substring(7)}`;
+
+        const processArgs = [SERVER_SCRIPT_PATH];
+        if (SERVER_SCRIPT_ARGS) {
+            SERVER_SCRIPT_ARGS.split(' ').forEach(argTemplate => {
+                // Replace potential {VAR} with values from env.
+                const arg = argTemplate.replace(/\{(\w+)\}/g, (match, key) => {
+                    return env.hasOwnProperty(key) ? env[key] : match;
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)